openmc/executor.py

```diff
@@ -5,6 +5,82 @@
 import openmc
 
 
+def process_CLI_arguments(volume=False, geometry_debug=False, particles=None,
```

Comment:
Probably should make this "private" with a leading underscore -- not something that I'd imagine a user calling